src/views/Dashboard.vue

```diff
@@ -49,25 +47,17 @@
               <v-icon medium>mdi-settings</v-icon>
             </v-list-item-avatar>
             <v-list-item-content>
-              <v-list-item-title class="title font-weight-thin">
-                Settings
-              </v-list-item-title>
-              <v-list-item-subtitle>
-                View your Hub permissions, and alter user preferences
-              </v-list-item-subtitle>
+              <v-list-item-title class="title font-weight-thin">Settings</v-list-item-title>
+              <v-list-item-subtitle>View your Hub permissions, and alter user preferences</v-list-item-subtitle>
```

Comment:
From what I could tell classes and tags are the same, so no need to change this one as well.